tests/wait-for-services.py

```diff
@@ -68,15 +66,6 @@ def check_mysql():
         conn.close()
 
 
-@try_until_timeout(Exception)
-def check_rediscluster():
-    test_host = REDISCLUSTER_CONFIG["host"]
-    test_ports = REDISCLUSTER_CONFIG["ports"]
-    startup_nodes = [{"host": test_host, "port": int(port)} for port in test_ports.split(",")]
-    r = rediscluster.StrictRedisCluster(startup_nodes=startup_nodes)
-    r.flushall()
```

Comment:
why isn't this needed anymore?

Reply:
I just did away with the waiting for the container to start - the startup time of installing the dependencies is more than enough time to cover it.